### PR TITLE
Unify redirections

### PR DESCRIPTION
**Pull request by ljgarcia**
Profile redirections were not always the same, this is an attempt to unify them

### CODE REVIEW

**Comment by ljgarcia** [comment]
@bedroesb if you come up with an idea on how to add this to a template or so, it would be great!

**Comment by bedroesb** [comment]
@ljgarcia haha I was looking into it on how to specify the redirect_from on a global level, but I did not find a solution. 

at this moment pages have the permalink without the backslash, an I tried to redirect from links with a backslash, but it appears that this redirect is not working on collection level, and can probably be taken out.

```yml
collections_dir: pages
collections:
  types:
    output: true
    permalink: /:collection/:path
    redirect_from:
      - "/:collection/:path/"
  meetings:
    output: true
    permalink: /:collection/:path
    redirect_from:
      - "/:collection/:path/"
  tutorials:
    output: true
    permalink: /:collection/:path
    redirect_from:
      - "/:collection/:path/"
```

I hope you scripted the redirects because this seemed like a lot of work :D

**Comment by ljgarcia** [comment]
It was a lot of copy&paste.
Those profiles that have a draft after a release are slightly different from the others that only have drafts or have a release as latest option

**Comment by bedroesb** [comment]
good job nevertheless!

File: pages/_profiles/Event/0.2-DRAFT-2019_06_14.html [hunk 1]
```diff
@@ -1,10 +1,15 @@
 ---
 redirect_from:
-- "devSpecs/Event/specification"
-- "devSpecs/Event/specification/"
-- "/devSpecs/Event/"
-- "/specifications/drafts/Event"
-- "/profiles/Event/"
+- "/devSpecs/Beacon/specification/"
```
Comment (AlasdairGray) [inline pages/_profiles/Event/0.2-DRAFT-2019_06_14.html:3]: Should these be Event?

File: pages/_profiles/Person/0.2-DRAFT-2019_07_19.html [hunk 2]
```diff
@@ -1,10 +1,15 @@
 ---
 redirect_from:
-- "devSpecs/Person/specification"
-- "devSpecs/Person/specification/"
-- "/devSpecs/Person/"
-- "/specifications/drafts/Person"
-- "/profiles/Person/"
+- "/devSpecs/Beacon/specification/"
```
Comment (AlasdairGray) [inline pages/_profiles/Person/0.2-DRAFT-2019_07_19.html:3]: Shouldn't these be Person?

File: pages/_profiles/Sample/0.2-RELEASE-2018_11_10.html [hunk 3]
```diff
@@ -1,11 +1,15 @@
 ---
 redirect_from:
-- "devSpecs/Sample/specification"
-- "devSpecs/Sample/specification/"
-- "/devSpecs/Sample/"
-- "/specifications/drafts/Sample"
-- "/specifications/Sample"
-- "/profiles/Sample/"
+- "/devSpecs/Beacon/specification/"
```
Comment (AlasdairGray) [inline pages/_profiles/Sample/0.2-RELEASE-2018_11_10.html:3]: Shouldn't these be Sample?

**Review by AlasdairGray — CHANGES_REQUESTED** [review]
I think there are few copy/paste errors

Really good job in making these consistent. I'd been slowly doing this as we had revisions of profiles.

File: pages/_profiles/Event/0.2-DRAFT-2019_06_14.html (hunk 1 above)
Comment (ljgarcia) [reply pages/_profiles/Event/0.2-DRAFT-2019_06_14.html]: Yes, fixed in latest commit

File: pages/_profiles/Person/0.2-DRAFT-2019_07_19.html (hunk 2 above)
Comment (ljgarcia) [reply pages/_profiles/Person/0.2-DRAFT-2019_07_19.html]: Yes, fixed in latest commit

File: pages/_profiles/Sample/0.2-RELEASE-2018_11_10.html (hunk 3 above)
Comment (ljgarcia) [reply pages/_profiles/Sample/0.2-RELEASE-2018_11_10.html]: Yes, fixed in latest commit

**Comment by ljgarcia** [comment]
Three profiles had the incorrect name in the redirection (Event, Person, Sample), now fixed.

**Review by AlasdairGray — APPROVED** [review]

**Pull request merged**